rms/search_engine/utils.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+
+class StopWordsProcessor:
```

Comment:
For now this is fine, but for the future we would like to cache the wordlist between instance creations